manifest/provider/datasource.go

```diff
@@ -130,7 +130,7 @@ func (s *RawProviderServer) ReadPluralDataSource(ctx context.Context, req *tfpro
 	if err != nil {
 		resp.Diagnostics = append(resp.Diagnostics, &tfprotov5.Diagnostic{
 			Severity: tfprotov5.DiagnosticSeverityError,
-			Summary:  "Failed to save resource state",
+			Summary:  "Failed to save resource state", // FIX ME
```

Comment:
Are these FIXMEs here to add a more meaningful summary?

Reply:
Those 4 don't reflect the errors they represent, I noticed this while working on this PR. I will open a new PR to fix it. Next time I will add more meaningful text that just **FIX ME**. 😔